CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ Changes since v2.16
 ### Fixes
 - Searching for applications by the original REMS generated id works, even if another id has been assigned. (#2564)
 - GA4GH Visa (output by the experimental /api/permissions API) timestamps are now in seconds, instead of milliseconds. (#2554)
+- The REMS `reset` command line command now works even when you have duplicate resource ids in the database. (#2557)
```

Comment:
Perhaps mention the missing constraint if going back to 2019 versions.

Reply:
I think this line is not exactly honest about what the change was :)

Reply:
not sure the public changelog is the right place for that, especially since we haven't documented rollbacks at all!

Reply:
but I can do it if you'd like